opendatahub-io/data-science-pipelines-operator
# add newlines between certs

## The issue resolved by this Pull Request:
Resolves https://issues.redhat.com/browse/RHOAIENG-11297

## Description of your changes:
If odh-trusted-ca-bundle has now newline, it concatenates the ca bundle with service-ca without any newline separate, an easy fix is to ensure there is always at least one new line between each ca bundle merge. 

## Testing instructions

Deploy podtopodtls in a self-signed environment with odh-trusted bundle with no newline at the end of file, run a pipeline, if it succeeds, it works.


Note that I have also adjusted the func tests to ignore trailing spaces, because it's incredibly buggy and non-intuitive to make work correctly, testing these portions should be saved for unit testing.

## Checklist
